src/app/tests/suites/certification/Test_TC_WNCV_2_1.yaml

```diff
@@ -50,6 +50,7 @@ tests:
           error: UNSUPPORTED_WRITE
 
     - label: "3b: reads back the RO mandatory attribute: Type"
+      disabled: true
```

Comment:
Why are tests being disabled?

Reply:
https://github.com/project-chip/connectedhomeip/issues/16995

Reply:
There is a problem with the constraint check generate template @bzbarsky-apple